replay_build_scripts/upload_build_artifacts.mjs

```diff
@@ -18,7 +19,8 @@ const BUILDKITE_BUILD_ID_ARTIFACT = "build_id";
 const BUILDKITE_ARTIFACT_DIRECTORY = path.join(
   process.env.BUILDKITE_BUILD_CHECKOUT_PATH || "./",
   "build_id",
-  currentPlatform()
+  currentPlatform(),
+  outputArchitecture()
```

Comment:
You need to update the `scripts/chromium/tests/buildkite_run_tests_from_metadata.sh` in backend for this change, as well as the test pipeline file as well.

Reply:
artifact, not metadata, sorry!

Reply:
Done over here https://github.com/replayio/backend/pull/7738#pullrequestreview-1427395314